frontend/packages/multicluster-sdk/src/api/search/useFleetSearchPoll.ts

```diff
@@ -1,22 +1,59 @@
 /* Copyright Contributors to the Open Cluster Management project */
-import { WatchK8sResource } from '@openshift-console/dynamic-plugin-sdk'
 import { useSearchResultItemsQuery } from './search-sdk'
 import { useMemo } from 'react'
-import { convertStringToQuery } from './searchUtils'
-import { SearchResult, UseMulticlusterSearchWatch } from './types'
+import { SearchResult, UseFleetSearchPoll } from './types'
 import { searchClient } from './search-client'
 
-export const useMulticlusterSearchWatch: UseMulticlusterSearchWatch = (
-  watchOptions: WatchK8sResource,
-  advancedSearch?: { [key: string]: string }
```

Comment:
Ugo removed this typing in his code, maybe we should scrutinize that decision--I don't know why it was made.